doc/api/packages.md

```diff
@@ -369,6 +369,45 @@ treating the right hand side target pattern as a `**` glob against the list of
 files within the package. Because `node_modules` paths are forbidden in exports
 targets, this expansion is dependent on only the files of the package itself.
 
+### Subpath folder mappings
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/35746
+    description: Runtime deprecation.
+  - version: v14.13.0
+    pr-url: https://github.com/nodejs/node/pull/34718
+    description: Documentation-only deprecation.
+-->
+
+> Stability: 0 - Deprecated: Use subpath patterns instead.
+
+Before subpath patterns were supported, a trailing `"/"` suffix was used to
+support folder mappings:
+
+```json
+{
+  "exports": {
+    "./features/": "./features/"
+  }
+}
+```
+
+_This feature will be removed in a future release._
+
+Instead, use direct [subpath patterns][]:
```

Comment:
I initially read this section and thought “why are we _adding_ a section to the docs for something we’re deprecating?” and only later realized you’re introducing the deprecated thing to then introduce what people should use instead. I worry that people might not get to the important part, which is at the end. Can we instead just introduce subpath patterns and this use case, as if the trailing slash version never existed?

Reply:
Yes I've added the feature to the docs to deprecate it :)

I could just remove the first example, would that help?

Otherwise I could leave out the section here and just have the deprecation but my concern then was users might miss that it is even deprecated if they aren't using the feature.

Reply:
At the very least I’d put the deprecated version last. But I think it would be better to just not include the deprecated info at all. Simply explain subpath patterns for the use case of exposing `*` within a folder, and why a user might want to do that, as if the deprecated feature had never existed.

Reply:
It would be nice if we could document `/` subpaths in the docs for older node versions tho, at least as a reference, even if we didn't include that info in docs for node versions in which it was deprecated. Is that possible?

Reply:
I've added a suggestion above, please take a look.

Reply:
As far as I can tell, for now the best compromise seems to be to leave this section as-is.

@ljharb @GeoffreyBooth if either of you have a direct suggestion please let me know and discuss further between yourselves, but this seems the best so far from the conversation unless you have further feedback.

Reply:
Thanks, I'm content with this for now; I'll definitely make suggestions in the future if I have ideas for improvement.